src/UKHO.ERPFacade.API/Controllers/S100SapCallbackController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace UKHO.ERPFacade.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class S100SapCallbackController : BaseController<S100SapCallbackController>
+    {
+        public S100SapCallbackController(IHttpContextAccessor contextAccessor) : base(contextAccessor)
```

Comment:
This should be called SapCallbackController. I know that it only processes S100 data at the moment, but that may not always be the case (similar to the EES callback endpoint).

Reply:
Resolved